app/scripts/controllers/pod.js

```diff
@@ -231,11 +133,23 @@ angular.module('openshiftConsole')
         }
       });
 
```

Comment:
Hmm. might be a little bit.  Doesn't look like you are using the `firstRun` flag anymore.